reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
+class Results extends Component {
 	render() {
-
-		var percent =(this.props.score / this.props.questions.length*100);
-		if (percent > 80){
-
-			var message ='Awesome Job!';
-		} else if(percent < 80 && percent >60)
-		{
-			var message = 'You did Ok !';
-		}
-		else {
-
-			var message ='Not satisfactory. Should try once again !';
+		const percent = (this.props.score / this.props.questions.length * 100 * 100);
+		let message;
+		if (percent > 80) {
+			message = 'Awesome Job!!';
+		} else if (percent > 60) {
+			message = 'You did Ok!';
+		} else {
```

Comment:
The code change looks good.  However, could you please update the success message to use two exclamation points instead of one and the failure message to use three exclamation points?